src/functionalTest/java/uk/gov/hmcts/reform/iacaseapi/CcdScenarioRunnerTest.java

```diff
@@ -335,6 +335,12 @@ private Headers getAuthorizationHeaders(Map<String, Object> scenario) {
                 .getHomeOfficeGenericAuthorization();
         }
 
+        if ("JudgeX".equalsIgnoreCase(credentials)) {
```

Comment:
Judge instead of JudgeX to be in line with the role and other roles we have used in the class.

Reply:
Resolved